bundle/libraries/same_name_libraries_test.go

```diff
@@ -0,0 +1,121 @@
+package libraries
```

Comment:
This seem to duplicate acceptance test? Or is there a case it's covering that I'm missing?